src/base-input.js

```diff
@@ -81,17 +81,7 @@ export default class BaseInput extends LitElement {
     ];
 
     /**
-     * @private
-     */
-    this.auroError = null;
-
-
-    const idLength = 36,
-      idSubstrEnd = 8,
-      idSubstrStart = 2;
-
-    /**
-     * @private
+     * @private Value is unique ID set at runtime
      */
     this.uniqueId = Math.random().
       toString(idLength).
```

Comment:
Super confused here. You removed all the logic that creates `idLength`? This variable is used in auro-input.js. 

I thought that this was weird, but when I rebased off of the latest merge of `next` and ran a build, the tests blew up pretty hard. 

What was the intent here?

Reply:
I see what happened. @jason-capsule42 with your update you refactored this pretty heavily https://github.com/AlaskaAirlines/auro-input/pull/49/files#diff-aedef65e662f6f161de4dda6e34b704053e1778b73199c5d58273e4d66c7f4fbR84-R98

what was the motivation to do this? 

Anyway, when I rebased @geoffrich branch over the updated `next` this is what happened. Can you two look overt this and let me know when this issue is resolved.